test/integration/pack/test_pack_docker.py

```diff
@@ -331,7 +331,7 @@ def test_image_specific_tarantool_versions(cartridge_cmd, project_without_depend
     project = project_without_dependencies
 
     tarantool_versions = [
-        {"input": "2", "expect": "Tarantool 2"},
+        {"input": "2", "expect": "Tarantool 2.11"},
```

Comment:
Even though there are no plans for Tarantool 2.12, I think "Tarantool 2" check is better here.